browser-entry.js

```diff
@@ -71,8 +71,8 @@ process.listenerCount = function (name) {
 
 process.on = function (e, fn) {
   if (e === 'uncaughtException') {
-    global.onerror = function (err, url, line) {
-      fn(new Error(err + ' (' + url + ':' + line + ')'));
+    global.onerror = function (msg, url, line, col, err) {
+      fn(err || new Error(msg + ' (' + url + ':' + line + ')'));
```

Comment:
[Feature] Since `col` is now available, I think it'd be nice to include it too:

```suggestion
      fn(err || new Error(msg + ' (' + url + ':' + line + ':' + col + ')'));
```